### ISSUE TITLE
Do the tabs on news.scripting.com work?

### ISSUE THREAD

**Issue by scripting** [scripting/Scripting-News#296]
Your help is much appreciated! 😄

Let's try this as a tab test.

https://news.scripting.com/

As with every other test...

1. Click a tab.
2. Ignore how long it takes to display the content.
3. Did it ignore your click?
4. Click another tab.
5. Do this a bunch of times.

Did it register your clicks or not?

Remember, forget everything else. 

All that matters is the tab clicking being heard by the software.

Thank you!

**Comment by richb-hanover** [comment]
> 3. Did it ignore your click?

No. ALL taps resulted in showing the new tab (tested on three tablet devices on news.scripting.com).

**Comment by scripting** [comment]
@richb-hanover -- as an aside, i guess this means the original problem you reported has now been solved and you don't have to use the lucky.wtf site? is that correct?

**Comment by richb-hanover** [comment]
I have stopped using (and testing against) the lucky.wtf site after you created the new test pages.

I will note that the Test2 site (http://scripting.com/code/testing/tabstest/) gives a perfect user experience, better than subsequent test pages. The GUI there shows that the tap was recognized by highlighting the tab immediately.

Newer test pages (http://scripting.com/code/testing/tabstest3/index.html and this morning, news.scripting.com) frequently fail to highlight the tab after a tap until a second or two have passed, making me think that I "missed the tap", and going to tap again.

The behavior on news.scripting.com is better than my original report because each tap is always recognized. But it is still not as good as Test 2. And one more report: my iPhone works flawlessly with these test pages - I only see this on my Android devices. Thanks for listening.

**Comment by scripting** [comment]
@richb-hanover -- there's not a whole lot of work to do to display an image! :-)

**Comment by richb-hanover** [comment]
And maybe that's my observation/question: Is it essential that the visual feedback (highlighting the background of the tapped tab) be held up until the contents have been fetched? 

PS I don't know much about this level of coding to know whether that behavior is even in your control.

PPS I am deeply aware that this question has soaked up a huge amount of your attention that you would have preferred to spend on Feedland and other high level projects.  Thank you for all this time

**Comment by burbridge** [comment]
![scriptingcom-tab-blank](https://github.com/scripting/Scripting-News/assets/3044969/8dd671a8-c946-4a81-bb27-93b2d0fd567b)

In macOS 14.4.1 (23E224), Safari Version 17.4.1 (19618.1.15.11.14), and in Safari Technology Preview release 191 (Safari 17.4, WebKit 19619.1.6.3) with extensions disabled:  
- Clicking on a tab activates the tab—but the contents are blank (please see attached screenshot). 
- Manually resizing the window sometimes restores the contents, but not always. 
- After the contents are restored by resizing the window, the “click to expand text” does not work. Clicking on an expansion dart expands . . . to blank. 
- Resizing the window via the green window button does not restore the contents.
- Refreshing the browser window does not restore the contents.

Everything works OK in Firefox.
